data/json/items/magazine/gas.json

```diff
@@ -36,5 +36,24 @@
     "count": 240,
     "flags": [ "NO_UNLOAD", "NO_RELOAD" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "weldgas": 240 }, "watertight": true } ]
+  },
+  {
+    "id": "co2_cartridge",
+    "looks_like": "pressurized_tank",
+    "type": "MAGAZINE",
+    "name": { "str": "carbon dioxide cartridge" },
+    "description": "A small aluminum cartridge designed for holding pressurized carbon dioxide.",
+    "weight": "100 g",
+    "volume": "260 ml",
+    "price": 600,
+    "price_postapoc": 100,
+    "material": [ "aluminum" ],
+    "symbol": "#",
+    "color": "light_gray",
+    "ammo_type": [ "co2" ],
+    "capacity": 1,
+    "count": 1,
+    "flags": [ "NO_UNLOAD", "NO_RELOAD" ],
```

Comment:
At least without a compressor (and means of generating or separating CO2).

Reply:
oxyacetylene is craftable, but all containers that store it have both of those flags too.